cfme/tests/containers/test_manageiq_ansible_providers.py

```diff
@@ -29,6 +29,7 @@ def ansible_providers():
     remove_tmp_files()
 
 
+@pytest.mark.skip(reason="Needs pzagal review")
```

Comment:
This shouldn't be in the code

Reply:
This the goal of this PR, skip broken tests.

Reply:
I don't understand what changes you expect to see in this PR, can you be clearer?

Reply:
I understand your point, but it should be uncollected and add TODO with that reason.

Reply:
@quarckster This code is not relevant, i redefine the reason, can you take a look?